### PR TITLE
Add pints, quarts and some volume unit unit tests

### PR DESCRIPTION
**Pull request by pstaabp**

### CODE REVIEW

File: htdocs/helpFiles/Entering-Units.html [hunk 1]
```diff
@@ -23,7 +23,7 @@ <h2 style="text-align: center">Units Available in WeBWorK</h2>
 	</tr>
 	<tr>
 		<td style="padding: 0.5rem; border: 1px solid black">Hours</td>
-		<td style="padding: 0.5rem; border: 1px solid black; text-align: center">hr</td>
+		<td style="padding: 0.5rem; border: 1px solid black; text-align: center">hr or hour or h</td>
```
Comment (Alex-Jordan) [inline htdocs/helpFiles/Entering-Units.html]: or `hours` or `hrs`

File: htdocs/helpFiles/Entering-Units.html [hunk 2]
```diff
@@ -55,7 +55,7 @@ <h2 style="text-align: center">Units Available in WeBWorK</h2>
 	</tr>
 	<tr>
 		<td style="padding: 0.5rem; border: 1px solid black">Meters</td>
-		<td style="padding: 0.5rem; border: 1px solid black; text-align: center">m</td>
+		<td style="padding: 0.5rem; border: 1px solid black; text-align: center">m or meters</td>
```
Comment (Alex-Jordan) [inline htdocs/helpFiles/Entering-Units.html]: or `meter` or `metre` or `metres`

File: lib/Units.pm [hunk 3]
```diff
@@ -230,11 +230,22 @@ our %known_units = (
 		factor => 0.0001,
 		m      => 3
 	},
+	# U.S./English volume units
 	cup => {
 		factor  => 0.000236588,
 		m       => 3,
 		aliases => ['cups']
 	},
+	pt => {
```
Comment (Alex-Jordan) [inline lib/Units.pm]: Judging by the adjacent entries (`cup` and `gallon`), maybe we have a standard where the singular full word is the key, and other things are aliases. So `pint => ... and `quart => ... `. And the plural full word is the first listed alias.

File: htdocs/helpFiles/Entering-Units.html [hunk 4]
```diff
@@ -109,6 +109,27 @@ <h2 style="text-align: center">Units Available in WeBWorK</h2>
...
+		<td style="padding: 0.5rem; border: 1px solid black">Deciliters</td>
+		<td style="padding: 0.5rem; border: 1px solid black; text-align: center">dL</td>
+	</tr>
+	<tr>
+		<td style="padding: 0.5rem; border: 1px solid black">Cups</td>
+		<td style="padding: 0.5rem; border: 1px solid black; text-align: center">cup</td>
+	</tr>
+	<tr>
+		<td style="padding: 0.5rem; border: 1px solid black">Pints</td>
+		<td style="padding: 0.5rem; border: 1px solid black; text-align: center">pt or pint</td>
```
Comment (Alex-Jordan) [inline htdocs/helpFiles/Entering-Units.html]: Do you want to list `pints`, `quarts`, and `gallons` here too?

File: htdocs/helpFiles/Entering-Units.html (hunk 2 above)
Comment (Alex-Jordan) [reply htdocs/helpFiles/Entering-Units.html]: Sorry, I thought you were adding the "re" variants here. Nevermind about that for this.

**Comment by pstaabp** [comment]
I'll make all these changes.  I had originally thought about not including plurals in the documentation, but I guess if we allow, I will.

File: htdocs/helpFiles/Entering-Units.html [hunk 5]
```diff
@@ -15,23 +15,23 @@ <h2 style="text-align: center">Units Available in WeBWorK</h2>
...
 	</tr>
 	<tr>
 		<td style="padding: 0.5rem; border: 1px solid black">Minutes</td>
-		<td style="padding: 0.5rem; border: 1px solid black; text-align: center">min</td>
+		<td style="padding: 0.5rem; border: 1px solid black; text-align: center">min, minute or minutes</td>
 	</tr>
 	<tr>
 		<td style="padding: 0.5rem; border: 1px solid black">Hours</td>
-		<td style="padding: 0.5rem; border: 1px solid black; text-align: center">hr</td>
+		<td style="padding: 0.5rem; border: 1px solid black; text-align: center">hr, h, hour or hours</td>
```
Comment (Alex-Jordan) [inline htdocs/helpFiles/Entering-Units.html]: It also accepts `hrs`. I'm not clear though if you want this to list every variant.

File: htdocs/helpFiles/Entering-Units.html [hunk 6]
```diff
@@ -191,11 +212,11 @@ <h2 style="text-align: center">Units Available in WeBWorK</h2>
 	</tr>
 	<tr>
 		<td style="padding: 0.5rem; border: 1px solid black">Angle degrees</td>
-		<td style="padding: 0.5rem; border: 1px solid black; text-align: center">deg</td>
+		<td style="padding: 0.5rem; border: 1px solid black; text-align: center">deg, º, degree or degrees</td>
```
Comment (Alex-Jordan) [inline htdocs/helpFiles/Entering-Units.html]: Should the special character be raw like this in source? I think we want it to be `&#x00B0;` here.

File: htdocs/helpFiles/Entering-Units.html (hunk 5 above)
Comment (pstaabp) [reply htdocs/helpFiles/Entering-Units.html]: I missed 'hrs'. I'm on the fence about adding all variants, but we should be consistent.

File: htdocs/helpFiles/Entering-Units.html (hunk 6 above)
Comment (pstaabp) [reply htdocs/helpFiles/Entering-Units.html]: Will switch.

**Comment by pstaabp** [comment]
Just made changes.  If close, let's merge and then everyone can discuss.  openwebwork/pg#859

**Pull request merged**